announcements/_posts/2016-05-04-week-14-letsencrypt.md

```diff
@@ -0,0 +1,15 @@
+---
+author: Justin W. Flory
+title: "Spring Week 14: LetsEncrypt"
+layout: post
+---
+
+Welcome to Week 14, RITluggers! Hope everyone's week is going well so far.
+
+In case you missed it at the last meeting, our new Eboard was officially elected at our last meeting. The 2016-2017 President is Justin W. Flory and Vice President is Solomon Rubin. For this week and the next, we'll be getting a headstart into running the club.
```

Comment:
The department of redundancy department would like to have a word with you about your first sentence ;)